### PR TITLE
Use sourcemap for development builds

### PR DESCRIPTION
**Pull request by djmitche**
This is an elaborate way of asking, "why `eval`"?

### CODE REVIEW

**Comment by helfi92** [comment]
`eval` => fast builds
`sourcemap` => slow builds

If we want source-maps for development, we should go with `eval-source-map`. Build speed is faster than  `sourcemap` but slower than `eval`. It's a trade off.

**Comment by eliperelman** [comment]
What @helfi92 says is correct, its a tradeoff between build speed and accuracy of line number source mapping during development.

https://webpack.js.org/configuration/devtool/#devtool

This just gave me another thought, we shouldn't stop generating sourcemaps in Travis so we get faster builds there.

File: .neutrinorc.js [hunk 1]
```diff
@@ -129,7 +129,7 @@ module.exports = {
   ],
   env: {
     NODE_ENV: {
-      development: ({ config }) => config.devtool('eval')
+      development: ({ config }) => config.devtool('sourcemap')
```
Comment (eliperelman) [inline .neutrinorc.js]: I'm not sure `sourcemap` is a valid value here, I think it's `source-map`. Regardless, we probably don't actually want this in development, but rather a different option from:

https://webpack.js.org/configuration/devtool/#development

`eval-source-map` is probably a good option.

**Comment by djmitche** [comment]
Hm, 'sourcemap' is working fine for me.  And it's not any slower than eval
was.

**Comment by eliperelman** [comment]
Resolved in #342.

**Pull request closed without merging**